.gitignore

```diff
@@ -7,3 +7,6 @@
 /testwrk/
 /wrk/
 /cc65.zip
+
+# Vim swap files
+.*.swp
```

Comment:
We only ignore files created by the tool chain(s) mentioned in the Makefiles. Not files created by random tools. Please revert your change to .gitignore.